specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json

```diff
@@ -2814,6 +2814,11 @@
           "type": "string",
           "description": "The name of the deployment."
         },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of the deployment."
+        },
```

Comment:
was the property recently added or was it simply missed while creating the swagger initially.